docs/source/glossary.md

```diff
@@ -47,4 +47,12 @@ Posterior
   Once you are satisfied with the model, posterior distribution can be summarized and interpreted. Common questions for the posterior include: intervals of defined boundaries, intervals of defined probability mass, and point estimates. When the posterior is very similar to the prior, the available data does not contain much information about a parameter of interest.
 
   - For more on generating and interpreting the posterior samples, see Statistical Rethinking 2nd Edition By Richard McElreath, chapter 3.
+
+Maximum a Posterior (MAP)
+  Similar to MLE, MAP is also utilized to estimate the parameters of models to explain the observed datasets. Unlike MLE, MAP tries to maximize the posterior function instead of the likelihood function.  
```

Comment:
When referencing MLE, we can link to the definition of MLE in this same glossary like this:
```
Similar to {term}`MLE`, MAP is also utilized...
```